docs/dev.md

```diff
@@ -0,0 +1,48 @@
+Development
+===========
+This page contains information for developers about how to contribute to this project.
```

Comment:
Copied from https://dans-knaw.github.io/dd-ingest-flow/dev/ but the intention looks broader than the content in both situations.